ambari_gem/spec/ambari_client_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe AmbariClient do
+  it 'has a version number' do
+    expect(AmbariClient::VERSION).not_to be nil
```

Comment:
Does this pass? If it does, then I guess I'm a Monkey's Uncle for my previous comment on version.